events/events.md

```diff
@@ -26,6 +26,7 @@ featured: true
 | December | **AAMT 2022** | |
 | 12 October | [**LoResMT 2022**](loresmt2022.md) | online |
 | 12 September | [**AMTA 2022**](amta2022.md) | Orlando, Florida |
+| 5-10 September | [MT Marathon](https://ufal.mff.cuni.cz/mtm22/) | Prague, Czech Republic |
```

Comment:
We could also create a new file called mtm2022.md, where we would have more details about this upcoming event and link to it.